third_party/wscript

```diff
@@ -35,4 +35,7 @@ def build(bld):
     if bld.env.bt_controller == 'cc2564x':
         bld.recurse('ti_bt_sp')
 
+    if bld.env.imu_sensor == 'lsm6dso':
+        bld.recurse('lsm6dso')
```

Comment:
for now: just use `if bld.is_asterix()`, we'll eventually make build system better using e.g. Kconfig+CMake